src/components/comments/Comment.js

```diff
@@ -1,10 +1,12 @@
 import React, { useContext, useRef } from "react"
 import { CommentContext } from "./CommentProvider"
+import { CommentEditForm } from "./CommentEditForm"
 import "./Comment.css"
 
 export const Comment = (props) => {
-    const {deleteComment} = useContext(CommentContext)
+    const {deleteComment, updateComment} = useContext(CommentContext)
```

Comment:
It looks like you are not using updateComment in this file, so just for cleaning up code purposes, I would get rid of this! Other than that it is awesome!

Reply:
@samanthajmaas Good catch! Fixed!